pkg/controllers/disruption/helpers.go

```diff
@@ -78,9 +79,7 @@ func SimulateScheduling(ctx context.Context, kubeClient client.Client, cluster *
 		pods = append(pods, n.reschedulablePods...)
 	}
 	pods = append(pods, deletingNodePods...)
-	scheduler, err := provisioner.NewScheduler(ctx, pods, stateNodes, pscheduling.SchedulerOptions{
-		SimulationMode: true,
-	})
+	scheduler, err := provisioner.NewScheduler(logging.WithLogger(ctx, operatorlogging.NopLogger), pods, stateNodes)
```

Comment:
Seems like you don't need the context elsewhere in this function for logging, so you could just set the loggin.WithLogger() for the context and pass it in here for L82 and L91 without having to do it two times

Reply:
I'd personally rather set it directly for now. If we were to add changes in later that started to log things, I could see it being potentially confusing for someone trying to reason about why they aren't getting any log output from the function call.